CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 - Remove decorations (dots at the beginning of each line) from output to stdout in admin commands.
 
+- Update `checks` to `3.3.0`, `cartridge` to `2.8.1`, `metrics` to `1.0.0` and
```

Comment:
I think commit message is a bit misleading. It's more like "app-dependency: update modules versions"